packages/dbos-cloud/login.ts

```diff
@@ -133,11 +26,15 @@ export async function login(host: string): Promise<number> {
     );
     const username = response.data as string;
     const credentials: DBOSCloudCredentials = {
-      token: token,
+      token: authResponse.token,
+      refreshToken: authResponse.refreshToken,
       userName: username,
     };
     writeCredentials(credentials)
     logger.info(`Successfully logged in as ${credentials.userName}!`);
+    if (getRefreshToken) {
+      logger.info(`Refresh token: ${authResponse.refreshToken}`)
+    }
```

Comment:
I think we'll have complains for printing this to stdout. Likely better in a file like the other.